js/service/accountservice.js

```diff
@@ -86,7 +86,14 @@ define(function(require) {
 			if (accounts.length > 1) {
 				accounts.add({
 					accountId: -1,
-					isUnified: true
+					isUnified: true,
+					id: btoa('all-inboxes'),
+					name: t('mail', 'All inboxes'),
+					emailAddress: t('mail', 'All inboxes'),
+					specialRole: 'inbox',
```

Comment:
🙈 so you're adding folder properties to the account?

Reply:
what do you mean? :p

Reply:
Obviously, the account does not have a property called `specialRole`. Only `Folder` objects do 😉

Reply:
Oh, okay! :)
I basically copied the property added into folders. I'm a bit lost with backbone & marionnette, feels harder than it could have been :p